benches/benchmarks/ipa_prove.rs

```diff
@@ -0,0 +1,42 @@
+use ark_std::rand::SeedableRng;
```

Comment:
This is a new benchmark to measure IPA proving time, which was missing in the benchmark set.